### PR TITLE
More translations support on health status page

### PR DESCRIPTION
**Pull request by ahinkle**
This PR adds a few translations for a little bit of customization on the health webpage.

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you take a look at the failing tests? You can run the tests locally with "composer test"

**Comment by ahinkle** [comment]
Sorry about that! All set 👍

**Comment by freekmurze** [comment]
Thanks!

File: resources/views/list.blade.php [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
         <div class="flex flex-wrap justify-center space-y-3">
-            <h4 class="w-full text-2xl font-bold text-center text-gray-900 dark:text-white">Laravel Health</h4>
+            <h4 class="w-full text-2xl font-bold text-center text-gray-900 dark:text-white">{{ __('health::notifications.laravel_health') }}</h4>
             <div class="flex justify-center w-full">
                 <x-health-logo/>
             </div>
             @if ($lastRanAt)
                 <div class="{{ $lastRanAt->diffInMinutes() > 5 ? 'text-red-400' : 'text-gray-400 dark:text-gray-500' }} text-sm text-center font-medium">
-                    Check results from {{ $lastRanAt->diffForHumans() }}
+                    {{ __('health::notifications.check_results_from') }} {{ $lastRanAt->diffForHumans() }}
```
Comment (mikemand) [inline resources/views/list.blade.php:17]: Should pass the date to the translation so right-to-left languages can also translate this without it being nonsensical.

**Pull request merged**